quodlibet/_main.py

```diff
@@ -206,9 +206,9 @@ def get_build_description():
 
         if build.BUILD_INFO:
             notes.append(build.BUILD_INFO)
-
-    version_string = ".".join(map(str, version))
-    note = " (%s)" % ", ".join(notes) if notes else ""
+    
+    version_string = u".".join(map(str, version))
```

Comment:
```suggestion
    version_string = ".".join(map(str, version))
```
Unicode literal syntax is long deprecates and Ruff now errors on this